docs/advanced/writing-declarations.md

```diff
@@ -512,7 +512,9 @@ declare module "creator" {
 
 Lua supports overloading of mathematical operators such as `+`, `-` or `*`. This is performed using the [metatable methods](https://www.lua.org/manual/5.4/manual.html#2.4) `__add`, `__sub`, `__mul`, `__div`, and `__unm`. Since TypeScript does not support operator overloading in its type system, this feature is hard to replicate. Unfortunately, this is not something that can be fixed properly right now without forking off our custom TypeScript version.
 
-However, there is a workaround that works decently: if you declare a type as an intersection type with `number`, it will inherit all mathematical operators. For example:
+However, there are workarounds that work decently.
+
+One method is to declare a type as an intersection type with `number`. It will then inherit all mathematical operators. For example:
```

Comment:
```suggestion
However, there are two possible workarounds. The first one is to declare a type as an intersection type with `number`. It will then inherit all mathematical operators. Keep in mind that this is only partially type safe and may require some additional casting.

Example:
```